Add the user attributes section and tenant domain infor section  to the try it out section in org login (#30)
wso2/docs-ciam-cloud: en/docs/guides/organization-login/try-organization-login.md

7. Click **Update** to save the configurations.

> 8.  Go to the User Attributes tab, and click on + Add User Attributes.

Reviewer: ```suggestion
8.  Go to the **User Attributes** tab, and click on + Add User Attributes.
```

> 8.  Go to the **User Attributes** tab, and click on + Add User Attributes.

Reviewer 2: ```suggestion
8.  Go to **User Attributes** and click on **+ Add User Attributes**.
```

>     - Select <i>Email, First Name, Last Name</i> and <i>Username</i> from the list.

Reviewer 2: Shall we add this as a separate step? step 9?

Author: Sure, I will update the document.

Author: Updated the PR.

>     <img src="../../../assets/img/guides/organization-login/try-it-out/app_add_user_attributes.png" alt="App User Attributes Configurations" width="700" style="border:1px solid grey">

Reviewer: Shall we remove the black background cover and show only the select user attributes part?

Author: Sure, I will change it.

> 9.  Click Save to update the configuration.

Reviewer 2: ```suggestion
9.  Click **Save** to add the user attributes and click **Update** to save all the configurations.
```

7. Click **Update** to save the configurations.

8.  Go to **User Attributes** and click on **+ Add User Attributes**.

> 9.  Select <i>Email, First Name, Last Name</i> and <i>Username</i> from the list..

Reviewer 2: ```suggestion
9.  Select `Email`, `First Name`, `Last Name`, and `Username` from the list of attributes.
```

    | Property  | Description   |
    |-----------|---------------|
    | WSO2IS_HOST   | The URL of the Identity Server.    |

>     | WSO2IS_TENANT_NAME   | Id of the Organization, that the created application resides in. <br> Default value for this proprty is ```carbon.super```   |

Reviewer 2: ```suggestion
    | WSO2IS_TENANT_NAME   | ID of the organization where the created application resides. <br> Default value for this property is `carbon.super`.   |
```

>     | WSO2IS_TENANT_NAME   | Id of the Organization, that the created application resides in. <br> Default value for this proprty is ```carbon.super```   |
>     | WSO2IS_CLIENT_ID  | The client ID obtained when creating an application on the Identity Server Console.    |
>     | WSO2IS_CLIENT_SECRET  | The client secret obtained when creating an application on the Identity Server Console.   |
>     | SAMPLE_ORGS   | The details of the organization. <br> `id`: ID of the sub-organizations using this shared application. |

Reviewer 2: ```suggestion
    | WSO2IS_TENANT_NAME   | ID of the organization where the created application resides. <br> Default value for this property is `carbon.super`.   |
    | WSO2IS_CLIENT_ID  | The client ID obtained when creating an application on the Identity Server Console.    |
    | WSO2IS_CLIENT_SECRET  | The client secret obtained when creating an application on the Identity Server Console.   |
    | SAMPLE_ORGS   | The details of the organization. <br> `id`: ID of the sub-organizations using this shared application. |
```

    ]
    ```

    !!! info

>         If the created application resides in a different organization, change the value in the ```WSO2IS_TENANT_NAME``` to that organization id.

Reviewer 2: If the users are just reading through the doc before testing this out, they might wonder where `WSO2IS_TENANT_NAME` is. 

We better add it to the sample.

Author: Sure, I will update the document

Author: Updated the PR.

    ]
    ```

    !!! info

>         If the created application resides in a different organization, change the value in the ```WSO2IS_TENANT_NAME``` to that organization id.

Reviewer 2: ```suggestion
        If the created application resides in a different organization, replace the `WSO2IS_TENANT_NAME` value with the respective organization ID.
```
